templates/README.md

```diff
@@ -13,6 +13,7 @@ Default: [`default`](./default.yaml) (⭐Ubuntu, with containerd/nerdctl)
 Distro:
 - [`almalinux-8`](./almalinux-8.yaml): AlmaLinux 8
 - [`almalinux-9`](./almalinux-9.yaml), `almalinux.yaml`: AlmaLinux 9
+- [`almalinux-kitten-10`](./almalinux-kitten-10.yaml): [experimental] AlmaLinux Kitten 10
```

Comment:
Experimental ones should be in the `experimental/` directory

Reply:
Right, well if CentOS Stream and Fedora are not "experimental"....